src/main/resources/application.properties

```diff
@@ -0,0 +1,23 @@
+# datasource PostgreSQl
+spring.sql.init.platform=postgres
+spring.datasource.url=jdbc:postgresql://localhost:5432/TechitEasy-2
+spring.datasource.username=postgres
+spring.datasource.password=Password
+spring.datasource.driver-class-name=org.postgresql.Driver
+server.port=8082
+# jpa
+spring.jpa.database=postgresql
+spring.jpa.show-sql=true
+
+# generate schema ddl to create tables
+spring.jpa.generate-ddl=true
+spring.jpa.hibernate.ddl-auto=create
```

Comment:
Voor het testen is het soms makkelijk om hier "create-drop" te gebruiken. Je krijgt dan wel wat warschuwingen bij het opstarten, maar die mag je negeren. Het grote voordeel van create-drop is dat je database elke herstart precies zo is zoals je in de data.sql beschrijft. Voor het inleveren van je eindopdracht zien we hier graag "update" staan, dat is wat je voor een (echte) productie omgeving ook zou gebruiken.